brew.sh

```diff
@@ -3,15 +3,30 @@
 # Install Brew if not installed
 command -v brew > /dev/null || /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-# Update Brew and Formulae
+# Update Brew and Formula
 brew update
 
 # Install brew bundle
 brew tap Homebrew/bundle
 
-# Install tools and apps from the Brewfile wih Brew and Cask
-brew bundle --file=Brewfile
-pecl install xdebug
+# Install tools and apps for Devolepers or Non-Developers
+echo "Type the job role ( Developer or Non-Developer ), followed by [ENTER]:"
+read role
```

Comment:
How would you run this in non-interactive mode? Shouldn't we read an argument or option from the CLI instead? This way we can just say "run this command" or write a script wrapped in an app than runs it without a prompt.

Reply:
@hkdobrev I really like your comment. Good call! 
Actually the idea was to use interactive mode. That is why the script asks "Type the job role ( Developer or Non-Developer ). Can I leave the implementation аs is or to creat two ./brew.sh like developer.sh and non-developer.sh ?

Reply:
@BaselAl My question is why the idea was to use interactive mode? If people run it themselves, they don't need that prompt. They either need one or other and for non-technical people is better to not have many prompts on the  command line.

Reply:
@hkdobrev Because I remember that we discussed something similar.

Reply:
@BaselAl I think when we've discussed it, I at least meant options (as in  command-line options) and you might have thought of options to manually choose. I even think, it could be 2 separate scripts that run a common one with a different argument/option. This way we could point a specific person to a specific file to run.